R/api-data-checker.r

```diff
@@ -0,0 +1,97 @@
+# This is a standalone function not run by the screener itself.
```

Comment:
Could you give an example of using the function / what format to feed files in with? (or move the checker example run to the top of the script?)